<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QMainWindow" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="signup_name_label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="signup_name_entry"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="signup_password_label">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="signup_loginID_label">
        <property name="text">
         <string>Login ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="signup_password_entry"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="signup_loginID_entry"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QTimeEdit" name="close_time_entry"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="gap_time_label">
        <property name="text">
         <string>Gap Time(Min)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="close_time_label">
        <property name="text">
         <string>Close Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="gap_time_entry"/>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="open_time_entry"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="open_time_label">
        <property name="text">
         <string>Open Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="signup_btn">
        <property name="text">
         <string>SignUp</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>signup_name_entry</tabstop>
  <tabstop>signup_loginID_entry</tabstop>
  <tabstop>signup_password_entry</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
